.eslintrc.json

```diff
@@ -62,7 +62,9 @@
     {
       "files": ["*.test.ts", "*.spec.ts"],
       "rules": {
-        "@typescript-eslint/no-non-null-assertion": "off"
+        "@typescript-eslint/no-non-null-assertion": "off",
+        "@typescript-eslint/no-unsafe-return": "off",
+        "@typescript-eslint/no-unsafe-call)": "off"
```

Comment:
is `no-unsafe-call)` having extra parenthesis at the end?

Reply:
nice catch, thanks!